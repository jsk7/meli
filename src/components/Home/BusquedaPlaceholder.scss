@import '../../styles/common';

.BusquedaPlaceholder {
  position: absolute;
  font-size: 1.2em;
  color: rgba(0,0,0, 0.5);
  @include vertical-align;
  @include horizontal-align;
}
