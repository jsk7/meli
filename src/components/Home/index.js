import React from 'react';
import './BusquedaPlaceholder.scss'

export default class BusquedaPlaceholder extends React.PureComponent {

  render() {
    return (
      <div className="BusquedaPlaceholder">
        ¿Que estás buscando?
      </div>
    );
  }
}
