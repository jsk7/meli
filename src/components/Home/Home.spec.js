import React from 'react';
import Home from './index';
import renderer from 'react-test-renderer';

describe('<Home />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Home />
    ).toJSON();
    expect(tree).toMatchSnapshot();

  });
});


