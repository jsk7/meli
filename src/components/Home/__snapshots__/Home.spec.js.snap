// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders correctly 1`] = `
<div
  className="BusquedaPlaceholder"
>
  ¿Que estás buscando?
</div>
`;
